### DISCUSSION TITLE
Image stitching fail

### DISCUSSION THREAD

**Discussion by wzl970722** [OpenStitching/stitching#180] [Q&A]
No match exceeds the given confidence threshold.
                                 Do your images have enough overlap and common
                                 features? If yes, you might want to lower the
                                 'confidence_threshold' or try another
                                 'detector'
![1](https://github.com/OpenStitching/stitching/assets/89897938/9061e29e-952a-4cf1-b114-920ced6872eb)
![2](https://github.com/OpenStitching/stitching/assets/89897938/f4621a26-db33-43c1-b9a3-2c26742a14be)

**Comment by lukasalexanderweber** [comment]
You need to stand still while you are taking the pictures

**Reply by wzl970722** [reply]
These two pictures look static. Can you help me try some methods to complete the stitching? Can you provide me with the parameters? 
The same goes for other pictures Thank you

**Reply by lukasalexanderweber** [reply]
Have you lowered the confidence threshold? And please use verbose to get more insights

**Reply by wzl970722** [reply]
from stitching import Stitcher
stitcher = Stitcher()
stitcher = Stitcher(detector="akaze", confidence_threshold=0.01)
panorama = stitcher.stitch_verbose(["/media/wzl/data1//test2/images/2/1.jpg","/media/wzl/data1//test2/images/2/2.jpg"])      
 The detector failed to use either method

**Reply by lukasalexanderweber** [reply]
As I said the images are not suitable

**Reply by wzl970722** [reply]
Why is it not suitable? It seems that there are many overlapping parts。。。

**Reply by lukasalexanderweber** [reply]
But the Rotation is wrong. As I said, you need to stand still and take pictures from left to right. In your sample Images you can clearly see that you moved to the right

**Reply by wzl970722** [reply]
So do we need to stand still and shoot from left to right

**Reply by lukasalexanderweber** [reply]
Yes
